yarn-project/end-to-end/src/e2e_epochs/epochs_l1_reorgs.test.ts

```diff
@@ -44,9 +48,24 @@ describe('e2e_epochs/epochs_l1_reorgs', () => {
     node = context.aztecNode;
     archiver = (node as AztecNodeService).getBlockSource() as Archiver;
     proverNode = context.proverNode!;
+
+    const account = privateKeyToAccount(generatePrivateKey());
+    l1ClientForMesssages = createExtendedL1Client(
+      [...test.l1Client.chain.rpcUrls.default.http],
+      account,
+      test.l1Client.chain,
+    );
+
+    const fundingTx = await test.l1Client.sendTransaction({
+      to: l1ClientForMesssages.account.address,
+      value: BigInt(1e16),
+    });
+
+    await test.l1Client.waitForTransactionReceipt({ hash: fundingTx });
   });
 
   afterEach(async () => {
+    await context.cheatCodes.eth.setIntervalMining(0);
```

Comment:
Is this needed anymore? It doesn't look like its being set to something else in `beforeEach`